### ISSUE TITLE
No button...

### ISSUE THREAD

**Issue by Ikeoh** [jesshmusic/fvtt-challenge-calculator#6]
No button for me...

![image](https://github.com/jesshmusic/fvtt-challenge-calculator/assets/34623527/30e59433-ad8c-4f65-9004-50a1493d7ec0)

---------------------------------

Foundry Virtual Tabletop: Version 11 Stable, 11.315
Game System: dnd5e, 3.0.1
Active Modules: 93
Performance Mode: 3

OS: Unknown
Client: FLC
GPU: ANGLE (NVIDIA, NVIDIA GeForce GTX 1050 Ti (0x00001C82) Direct3D11 vs_5_0 ps_5_0, D3D11)
Max Texture Size: 16384

Scene: 6000 x 4600 | Grid Size: 100 | Padding Percentage: 0.25
Walls: 24 | Ambient Lights: 1 | Ambient Sounds: 0 | Tiles: 36 | Tokens: 3

Actors: 30 | Items: 40 | Journal Entries: 157 | Rollable Tables: 60 |
Playlists: 94 | Compendium Packs: 69 | Chat Messages: 0

**Comment by jesshmusic** [comment]
Any errors? It also only shows for the default sheet or the default npc sheet. And lastly it only shows for GM players. 
If still not showing it might be an error in the chat log?

**Comment by Ikeoh** [comment]
![image](https://github.com/jesshmusic/fvtt-challenge-calculator/assets/34623527/eeec49ef-15eb-41c7-ac51-f88f5546bd58)
![image](https://github.com/jesshmusic/fvtt-challenge-calculator/assets/34623527/db63cd84-6756-4f11-a961-8a48289937a2)

**Comment by jesshmusic** [comment]
Ok that error is a pain, I will investigate. (DC 20 INT)

**Comment by thepting** [comment]
Yeah, I have the same problem with no button appearing.  Is this because of the new DnD 3.0.x Game System changing the sheets?  I have some other modules that don't work on the new default NPC sheets.

**Comment by jesshmusic** [comment]
This module is now updated for v13
